tidy3d/web/api/asynchronous.py

```diff
@@ -16,14 +16,14 @@ def run_async(
     simulation_type: str = "tidy3d",
     parent_tasks: Dict[str, List[str]] = None,
 ) -> BatchData:
-    """Submits a set of Union[:class:`.Simulation`, :class:`.HeatSimulation`] objects to server,
+    """Submits a set of Union[:class:`.Simulation`, :class:`.HeatSimulation`, :class:`.EMESimulation`] objects to server,
```

Comment:
This is maybe something we should add later, but I believe we would benefit from having a `SolverSimulationType` definition, also in the context of 3.0, so that all these methods can just use that type both at the docstring and parameter typing check level @tylerflex